apps/spectator/app/game/[id]/_components/Lineup/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { useGameLineupById } from '@/queries/useGameLineupById';
+
+import * as styles from './Lineup.css';
+import LineupPlayerList from './PlayerList';
+import LineupTeam from './Team';
+
+type LineupProps = {
+  gameId: string;
+};
+
+export default function Lineup({ gameId }: LineupProps) {
+  const {
+    data: [firstTeam, secondTeam],
+  } = useGameLineupById(gameId);
```

Comment:
멀티라인 suggest 방법 찾았습니다ㅋㅋ
사소한 부분이긴 한데 구조분해를 이중으로 하게 되면 오히려 가독성을 해친다는 느낌을 받아서 개인적으로 아래와 같은 방법을 선호해요.

```suggestion
  const { data: lineups } = useGameLineupById(gameId);
  const [firstTeam, secondTeam] = lineups
```

이렇게 하면 아마 prettier 글자 수도 초과하지 않아서 두 줄로 줄일 수 있을 거 같아요. 물론 굉장히 취향의 영역이다 보니 무시하셔도 좋습니다!

Reply:
ㅋㅋㅋ 저도 작성할 때 고민했던 부분인데, 저는 개인적으로 제가 썼던 방식처럼 쭉 연달아 뽑아내는게 흐름이 끊기지 않아서 파악하기 쉽더라구요. 이 부분은 굳이 고치진 않아도 되니까 참고만 하도록 하겠습니다 좋은 의견 감사합니다 ㅎㅎ